Dockerfile_test_py310

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:20.04
+
+RUN apt update
+RUN ln -fs /usr/share/zoneinfo/Europe/Prague /etc/localtime
```

Comment:
Should remove this, can stick to UTC (which I believe is the default)

Reply:
Good point with the UTC, I will change it. But the default zone needs to be linked before installing build-essential to avoid apt prompting.